src/mach/constants.rs

```diff
@@ -351,6 +351,27 @@ pub mod cputype {
             }
 
             /// Get the architecture name from cputype and cpusubtype
+            ///
+            /// When using this method to determine the architecture
+            /// name of an instance of
+            /// [`goblin::mach::header::Header`](/goblin/mach/header/struct.Header.html),
+            /// use the provided method
+            /// [`cputype()`](/goblin/mach/header/struct.Header.html#method.cputype) and
+            /// [`cpusubtype()`](/goblin/mach/header/struct.Header.html#method.cpusubtype)
+            /// instead of corresponding field `cputype` and `cpusubtype`.
+            ///
+            /// For example:
+            ///
+            /// ```rust
+            /// use std::fs::read;
+            /// use goblin::mach::constants::cputype::get_arch_name_from_types;
+            /// use goblin::mach::Mach;
+            ///
+            /// let buf = read("path/to/macho").unwrap();
```

Comment:
I believe this is causing CI failure. I’d suggest to use and_then in example so unwrap doesn’t fall or experiment at using one of the checked in macho binaries. Former probably better for example.